apps/bixarena/api/src/main/java/org/sagebionetworks/bixarena/api/model/entity/ExternalAccountEntity.java

```diff
@@ -0,0 +1,73 @@
+package org.sagebionetworks.bixarena.api.model.entity;
+
+import jakarta.persistence.*;
+import java.time.OffsetDateTime;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+@Entity
+@Table(
+  name = "external_account",
+  uniqueConstraints = {
+    @UniqueConstraint(columnNames = { "user_id", "provider" }),
+    @UniqueConstraint(columnNames = { "provider", "external_id" }),
+  }
+)
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString(exclude = { "user" })
+public class ExternalAccountEntity {
```

Comment:
Adapted from OC, where external accounts can be Google or Synapse accounts. Currently, only Synapse account are supported as external account to login in BixArena.